src/MSBuild/XMake.cs

```diff
@@ -1761,6 +1761,18 @@ private static bool PrintTargets(string projectFile, string toolsVersion, Dictio
                         MessageImportance.Low));
             }
 
+            NativeMethodsShared.SAC_State SAC_State = NativeMethodsShared.GetSACState();
+            if (SAC_State != NativeMethodsShared.SAC_State.NotApplicable || SAC_State != NativeMethodsShared.SAC_State.Missing)
+            {
+                messages.Add(
+                    new BuildManager.DeferredBuildMessage(
+                        ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword(
+                            "SAC",
+                            ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword(
+                                "SAC_" + SAC_State.ToString())),
+                        MessageImportance.Low));
```

Comment:
In ideal case the evaluation state leads to slightly different message (somehting like "Detected that Smart App Control feature is in evaluation mode. It is recommended to turn this of in development environemnt as otherwise performance might be impacted.") that would be of a high priority

Reply:
Can we have it as a `MessageImportance.High` for the eval? :-)